frontend/web/components/pages/ChangeRequestPage.js

```diff
@@ -154,7 +154,9 @@ const ChangeRequestsPage = class extends Component {
     const isScheduled =
       new Date(changeRequest.feature_states[0].live_from).valueOf() >
       new Date().valueOf()
-    const scheduledDate = moment(changeRequest.feature_states[0].live_from)
+    const scheduledDate = changeRequest.feature_states
+      ? moment(changeRequest.feature_states[0].live_from)
```

Comment:
Is this logic correct? I think it doesn't align with the PR description since it prioritizes the feature_states over the environment_feature_versions:
> Use the live_from value from the first environment feature version instead of the feature state if it exists.

Reply:
A change request can only have either `feature_states` or `environment_feature_versions` so the result will be the same whichever way round we have it.

Reply:
Oh Ok, then it will work. Approved

Reply:
I actually did have to swap them around since the FE pulls the feature_states out of the environment_feature_versions but the feature_states do not have the live_from attribute.

Reply:
LGTM